src/lib/web-component/log-view/log-view-element.js

```diff
@@ -22,12 +22,16 @@ class LogView extends HTMLElement {
     shadow.appendChild(logTableInstance);
   }
 
+  set modelFilter(filterFunc) {
```

Comment:
Let's merge `set modelFilter` with `filterLogViewItems`, and choose `setLogFilter` as the method name.

Reply:
done :-)